docs/site/Calling-soap-web-services.md

```diff
@@ -0,0 +1,191 @@
+---
+lang: en
+title: 'Calling SOAP Web services'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, REST, SOAP
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Calling-soap-web-services.html
+---
+
+To get started with accessing SOAP Web services you need to create a datasource,
```

Comment:
Similar suggestion as in the "calling REST APIs" page:
```suggestion
To get started with accessing SOAP Web services, you need to create a datasource
```

Reply:
Done